### ISSUE TITLE
It says 4 issues in materialize css file and wont load properly in live server

### ISSUE THREAD

**Issue by Isonomic** [iamshaunjp/pwa-tutorial#2]
It says 4 issues in materialize css file and wont load properly in live server

**Comment by AbePlays** [comment]
Yeah, don't worry about them. The issues are mainly because the code editor is not able to identify spacing / patterns in the code. The minified file removes whitespaces, strips comments, and optimizes/shortens a few common programming patterns.
